src/main/java/appeng/client/render/cablebus/CubeBuilder.java

```diff
@@ -251,6 +257,51 @@ private UvVector getDefaultUv( EnumFacing face, TextureAtlasSprite texture, floa
 		return uv;
 	}
 
+	private UvVector getStandardUv( EnumFacing face, TextureAtlasSprite texture, float x1, float y1, float z1, float x2, float y2, float z2 )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00107)